DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/TopCourses/Index.cshtml

```diff
@@ -1,5 +1,7 @@
-﻿@inject IConfiguration Configuration
+﻿@inject IConfiguration configuration
+@using DigitalLearningSolutions.Web.Models.Enums
 @using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.TopCourses
+@using Microsoft.AspNetCore.Mvc.TagHelpers

```

Comment:
Think we can get rid of configuration here, and the unused import that was introduced.